src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -686,17 +695,59 @@ def load_dichotomous_exposure(
 def load_dichotomous_excess_shift(
     location: str, distribution_data: Tuple, is_risk: bool
```

Comment:
You are not using the `is_risk` value anywhere anymore. Is it always False? If so, you should remove the parameter entirely. If it is not always False, then you've introduced a bug here.